src/data/registry.xml

```diff
@@ -415,6 +438,14 @@
       <dimensions>horizontal_dimension vertical_layer_dimension</dimensions>
       <initial_value>zvir</initial_value>
     </variable>
+    <variable local_name="cp_or_cv_dycore"
+              standard_name="specific_heat_of_air_used_in_dycore"
+              units="J kg-1 K-1" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <long_name>Enthalpy or internal energy scaling factor for energy consistency</long_name>
```

Comment:
While long name is correct I am not sure I would understand exactly what it is without documentation.

Perhaps change long name to:

specific heat of air used in the dynamical core (enthalpy for pressure-based dynamical cores and internal energy for z-based dynamical cores)

Reply:
Thanks, updated!